.github/workflows/release.yml

```diff
@@ -0,0 +1,55 @@
+name: Automated Version Branching
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Version number for the release (e.g., 1.0.0)"
+        required: true
+        type: string
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.ref }}  # This makes sure we check out the branch where the workflow runs
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.10"
+
+      - name: Install Only Specified Dependencies
+        run: |
+          python -m venv venv
+          source venv/bin/activate
+          pip install --upgrade pip
+          pip install -r ./shimming-toolbox/requirements_st.txt
+          pip install -r ./fsleyes-plugin-shimming-toolbox/requirements_stplugin.txt
```

Comment:
I think that installing Shimming Toolbox in a different way than the provided way of installing it (make install) is bound to create issues. As an example right now, fsleyes and its requirements are not in the pinned dependencies list. Unless there was another reason I'm not thinking, we should probably use "make install".

I also just noticed that fsleyes seems to be on [Pypi](https://pypi.org/project/fsleyes/). We should include it in the list of dependencies and remove it from the conda install of the installer.

Reply:
Fixed with [900b0cc](https://github.com/shimming-toolbox/shimming-toolbox/pull/585/commits/900b0cc221fd14c2e42ed02ed38828a9f565fa8f) and [e08ba2f](https://github.com/shimming-toolbox/shimming-toolbox/pull/585/commits/e08ba2fe4edef2d44afdba1eb5bc097f5752b2f4)

Reply:
I think we should do a separate PR for fsleyes. I can make it after this PR is closed.

Reply:
I opened an issue (#588)